config/config.go

```diff
@@ -89,6 +89,8 @@ func (config *Config) parseFlags() {
 
 	// SSE
 	flag.BoolVar(&config.SSE.Enabled, "sse-enabled", false, "Enable sse event stream.")
+	flag.IntVar(&config.SSE.Retries, "sse-retries", 0, "Number of times to recover SSE stream.")
+	flag.IntVar(&config.SSE.RetryBackoff, "sse-retry-backoff", 0, "Configuration of initial time between retries to recover SSE stream.")
```

Comment:
Correct we should use duration here.

```
		seconds := time.Duration(i * s.retryBackoff)
		time.Sleep(seconds * time.Second)
```

https://github.com/allegro/marathon-consul/blob/e36ce2dc17344704e6afb217f5f103c83c585264/marathon/streamer.go#L68-L69